libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -24,23 +24,32 @@ import {
     Component,
     ContentChild,
     EventEmitter,
+    forwardRef,
     Input,
     OnDestroy,
     OnInit,
     Optional,
     Output,
-    TemplateRef,
-    ViewEncapsulation,
     Provider,
-    forwardRef
+    TemplateRef,
+    ViewChildren,
+    ViewEncapsulation
 } from '@angular/core';
-import { ControlContainer, FormGroup, AbstractControl } from '@angular/forms';
+import { AbstractControl, ControlContainer, FormGroup } from '@angular/forms';
 import { Subject } from 'rxjs';
 import { coerceBooleanProperty } from '@angular/cdk/coercion';
 
 import { FormField } from '../form-field';
 import { FormGroupContainer } from '../form-group';
-import { LabelLayout, HintPlacement, FormZone } from '../form-options';
+import { FormZone, HintPlacement, LabelLayout } from '../form-options';
```

Comment:
As I already commented in form-field.ts do we need formZones. Is this in new Design? We have columns instead no?

Reply:
Removed